time stamp
float64 gpstime

float64 longitude
float64 latitude
float64 height

float64 heading
float64 pitch
float64 roll

float64 AngRateRawX
float64 AngRateRawY
float64 AngRateRawZ

float64 AccelRawX
float64 AccelRawY
float64 AccelRawZ
